src/components/presentational/DumbbellChart/DumbbellChart.tsx

```diff
@@ -0,0 +1,96 @@
+import "./DumbbellChart.css";
+import line from './horizontalLine.svg';
+import { useEffect, useState } from "react";
+import { debounce } from "lodash";
+import MyDataHelps from "@careevolution/mydatahelps-js";
+import Dumbbell from "./Dumbbell";
+import React from "react";
+
+export interface ClosedInterval { values : number[]};
+
+export interface DataPoint {
+    dataSet1 : ClosedInterval;
+    dataSet2 : ClosedInterval;
+}
+
+export interface Dumbbell {
+    dataPoint? : DataPoint;
+    xValue : string,
+    class? : DumbbellClass 
+}
+
+export interface Axis {
+    yRange : ClosedInterval;
+    yIncrement : number;
+    xIncrement : number;
```

Comment:
This is something that should be figured out automatically from the data and should not be specified in px...your chart needs to fill the screen and not just be anchored to the left side on large screens.  Would look at the other charts in this library and how they space out dynamically as you stretch the screen - basically x-axis things should be specified in % and not in px

This is a good example: https://ui.mydatahelps.org/?path=/story/symptomshark-presentational-overallexperiencechart--default

![Screenshot 2023-11-14 at 10 27 44 AM](https://github.com/CareEvolution/MyDataHelpsUI/assets/1643171/96adeaae-7320-466b-bf5b-6893ec4300a7)